Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/PowerTransmissions/GearBox.cs

```diff
@@ -730,7 +729,7 @@ public void Update(float elapsedClockSeconds)
         }
     }
 
-    public enum TypeGearBox
+    public enum TypesGearBox
```

Comment:
For this enum, you can also name it GearBoxTypeType or GearBoxTypeEnum.

Reply:
Thanks for the suggestion.

The PR is still in draft, so I will respond to the other comments when I am happy to seek approval.

I have already taken onboard some of the suggestions.

Thanks